islands/CounterCard.tsx

```diff
@@ -51,8 +51,33 @@ export default function Counter(props: SharedProps) {
   const [timer, setTimer] = useState(0);
   const ipc = new BroadcastChannel("counterIpc");
 
+  const THRESHOLD_CLICKS = 10; // Maximum number of clicks in an interval
```

Comment:
I think its better if we do it every 25 clicks so it doesn't feel too strict

Reply:
Just changed the value to `30`